lib/checks/report/report.rb

```diff
@@ -0,0 +1,28 @@
+module Checks
+  module Report
+    class Report
```

Comment:
The alternative I was thinking of is something like -

```ruby
class SomeCheck
  HEADERS = %w(...)

  def run(...)
    ...
    results
 end
end
```

With the Report being constructed by the caller. I'm fine with addressing this in a separate PR (or not at all) though.